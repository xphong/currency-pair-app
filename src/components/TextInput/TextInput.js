import React from 'react';

import styles from './TextInput.css';

const TextInput = props => {
  return <input className={styles.input} type="text" {...props} />
}

export default TextInput;
