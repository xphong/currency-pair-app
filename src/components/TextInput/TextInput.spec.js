import React from 'react';

import TextInput from './TextInput';

describe('TextInput Component', () => {
  it('should render TextInput', () => {
    const wrapper = shallow(<TextInput />);

    expect(wrapper).toMatchSnapshot();
  });

  it('should render TextInput with props', () => {
    const props = {
      value: 'Goku',
      name: 'Kakarot'
    }
    const wrapper = shallow(<TextInput {...props} />);

    expect(wrapper).toMatchSnapshot();
  });
});
