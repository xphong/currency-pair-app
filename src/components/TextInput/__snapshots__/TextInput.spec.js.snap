// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInput Component should render TextInput 1`] = `
<input
  className="input"
  type="text"
/>
`;

exports[`TextInput Component should render TextInput with props 1`] = `
<input
  className="input"
  name="Kakarot"
  type="text"
  value="Goku"
/>
`;
